pact/Pact/Core/IR/Eval/Runtime/Utils.hs

```diff
@@ -97,7 +97,8 @@ lookupFqName fqn =
 getDefCap :: (MonadEval b i m) => i -> FullyQualifiedName -> m (EvalDefCap b i)
 getDefCap info fqn = lookupFqName fqn >>= \case
   Just (DCap d) -> pure d
-  _ -> failInvariant info "Expected DefCap"
+  Just _ -> failInvariant info "Expected DefCap"
+  _ -> failInvariant info "Expected DefCap; got nothing"
```

Comment:
:+1: we should go over all the `failInvariant` invocations and enhance the messages.

Reply:
Yep, that's on my todo list as a separate issue. We'll need to have a proper ADT for various invariant failures instead of just strings as they are now.

Reply:
Indeed. I agree, we need a better ADT for invariant failures.